test.cpp

```diff
@@ -1,38 +1,74 @@
 #include "shift_proposal.h"
 
 #include <cassert>
+#include <array>
+#include <deque>
 #include <forward_list>
+#include <list>
 #include <vector>
 
+namespace {
+    template<class Sequence>
+    void test() {
+        using std::begin;
+        using std::end;
+
+        constexpr int n = 5;
+
+#ifdef __clang__
+// Tell clang to shut up about std::array {} initializations.
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wmissing-braces"
+#endif // __clang__
+
+        // shift left by i in [-2n, 3n)
+        for (auto i = -2 * n; i < 3 * n; ++i) {
+            auto const clamped = std::clamp(i, 0, n);
+            Sequence s{{1, 2, 3, 4, 5}};
```

Comment:
@CaseyCarter Since you disabled the warning, I guess you meant to keep this single-braced? (seeing as the rest of the test cases' Sequence objects are still single-braced)